### PR TITLE
[Testing] Absolute Roleplay

### PR DESCRIPTION
**Pull request by serifas**

### CODE REVIEW

**Comment by serifas** [comment]
Just checking in to see what the status is on the plugin code review.
Are there any updates as of yet?

**Comment by reiichi001** [comment]
Testing results:
- I would show a warning if the passwords do not match during registration. Currently, the plugin does not react. (the same applies to the forget password window)
- If a user closes the verification window, there's no way to get it back. The plugin will just say the email is already in use.
- A user can submit a forgot password before they have verified their account. (Is this intentional?)
- After submitting information for account restoration, the window does not go away. It also has cut-off text. 
image

- The wifi button by connected does not appear to do anything. Can you elaborate on what this is for?
- the DTRBar entry might not actually be by the map. I would suggest rewording this to "server info bar" or "top information bar" or similar.
- While logged in, the connections button does not seem to do anything

**Comment by serifas** [comment]
I will update the source with the changes. Just give me until probably tomorrow to submit it. Thank you for updating me.

**Comment by serifas** [comment]
also the way I have the registration setup with verification is working as I want it to asside from the message not displaying if passwords don't match, and the sizing issue.
The restoration window is intended to allow the user to request multiple times so if they close it they can just submit another request. 
The connection button is for if the server goes down the users can manually reconnect. It doesn't really do anything if the server connection hasn't died.
As far as the DTRBar goes, are you talking about the hint I gave in the command to open the window?
I am fixing the window sizing and password messages tho. but the rest I believe is working correctly.

**Comment by serifas** [comment]
is this still being reviewed?

**Comment by reiichi001** [comment]
> is this still being reviewed?

It's still making its way through review. The PAC member assigned to code review has not had a lot of availability due to IRL taking priority. (And other PAC members who can do the code review portion have also been pretty busy.) I'll mention this in our internal review sections.

**Comment by serifas** [comment]
thank you, sorry, I didn't mean to come off as pestering <3

**Comment by goaaats** [comment]
bleatbot, rebuild

**Comment by reiichi001** [comment]
bleatbot, rebuild

**Comment by reiichi001** [comment]
Apologies on the long wait! Code review on this plugin is complete. I'm waiting on a couple final checks, but this looks on track to be merged!

**Comment by serifas** [comment]
thank you soo soo much. I am soo hyped

**Comment by serifas** [comment]
also I don't know whether this matters but the pending-rule-compliance status is still there for me but completed-rule-compliance is too.

**Comment by reiichi001** [comment]
> also I don't know whether this matters but the pending-rule-compliance status is still there for me but completed-rule-compliance is too.

I was using that to mark the hold. I haven't heard anything to the contrary though, so we're doing it!

Congratulations on your first plugin added to the main Dalamud plugin repository!

As a first-time contributor, if you're in the `XIVLauncher & Dalamud` Discord server and would like to get access to the private developer channels for plugin feedback and others, please let us know your Discord handle and we'll grant you the role.

**Review by reiichi001 — APPROVED** [review]

**Comment by serifas** [comment]
thank you soo soo much! also if you see this before Kaz does, please add the role in discord to me if possible.

**Comment by serifas** [comment]
discord user = brattyfae
